dashboard/app/controllers/plc/enrollment_task_assignments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Plc::EnrollmentTaskAssignmentsController < ApplicationController
```

Comment:
`before_action :set_plc_enrollment_task_assignment` is now redundant -- `load_and_authorize_resource` will create the instance variables.